standard-covoiturage_openapi.yaml

```diff
@@ -756,6 +951,84 @@ components:
           type: string
           description: Brand of the car.
 
+
+    Booking:
+      type: object
+      required:
+        - id
+        - status
+        - price
+        - passengerPickupDate
+        - passengerPickupLat
+        - passengerPickupLng
+        - passengerDropLat
+        - passengerDropLng
+      properties:
+        id:
+          $ref: '#/components/schemas/bookingId'
+        passengerPickupDate:
+          type: number
+          description: Passenger pickup datetime as a UNIX UTC timestamp in seconds.
+          format: long
+        passengerPickupLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger pick-up point.
+        passengerPickupLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger pick-up point.
+        passengerDropLat:
+          type: "number"
+          format: "double"
+          description: Latitude of the passenger drop-off point.
+        passengerDropLng:
+          type: "number"
+          format: "double"
+          description: Longitude of the passenger drop-off point.
+        passengerPickupAddress:
+          type: string
+          description: String representing the pickup-up address.
+        passengerDropAddress:
+          type: string
+          description: String representing the drop-off address.
+        status:
+          $ref: '#/components/schemas/bookingStatus'
+        duration:
+          type: integer
+          description: Carpooling duration in seconds.
+        distance:
+          type: integer
+          description: |
+            Carpooling distance in meters. When the booking is COMPLETED or VALIDATED, this is the actual distance travelled if available.
+        webUrl:
+          type: string
+          description: URL of the booking on the webservice provider platform.
+        price:
+          $ref: '#/components/schemas/Price'
+        car:
+          $ref: '#/components/schemas/Car'
+        message:
+          type: string
+          maxLength: 500
+          description: Free text content of a message. The message can contain all the details (phone number, email, etc.) allowing the recipient to call back the sender in order to carpool with him/her.
```

Comment:
Since `Booking` is no more inherited from `ConnectionMessage`, let's change the description of the `Booking.message` attribute to simply :  

> Free text content of a message.

The ~The message can contain all the details (phone number, email, etc.) allowing the recipient to call back the sender in order to carpool with him/her.~ part car be removed because irrelevant in case of a Booking.